account_avatax/__manifest__.py

```diff
@@ -1,7 +1,8 @@
 {
-    "name": "Avalara Avatax Connector",
+    "name": "Taxes using Avalara Avatax API",
     "version": "14.0.1.0.0",
-    "author": "Fabrice Henrion, Odoo Community Association (OCA)",
+    "author": "Open Source Integrators, Fabrice Henrion, Odoo SA,"
```

Comment:
Please keep Fabrice as the author.
There is some back and forth email history about this.

Reply:
I mean, remove OSI and Odoo from the authors. Thanks.